Editor/Elements/Graph/BasePort.cs

```diff
@@ -139,11 +139,18 @@ public class BasePort : VisualElement, IPositionable {
             get => portColor;
             set {
                 portColor = value;
-                UpdateCapColor();
+                CapColor = portColor;
+                m_ConnectorBox.style.unityBackgroundImageTintColor = portColor;
             }
         }
         public virtual Color DefaultPortColor { get; } = s_DefaultColor;
-        public virtual Color DisabledPortColor { get; } = s_DefaultDisabledColor;
+        public virtual Color DisabledPortColor {
+            get {
+                Color color = PortColor * 0.3f;
```

Comment:
can we avoid this magic number (0.3f) here somehow? :)

Reply:
Maybe it could be some constant?

```csharp
private const float k_DisabledColorMultiplier = 0.3f;
```